src/main/java/net/sf/jabref/shared/prefs/SharedDatabasePreferences.java

```diff
@@ -1,18 +1,29 @@
 package net.sf.jabref.shared.prefs;
 
+import java.io.UnsupportedEncodingException;
+import java.security.GeneralSecurityException;
 import java.util.Optional;
 import java.util.prefs.BackingStoreException;
 import java.util.prefs.Preferences;
 
-import net.sf.jabref.Globals;
```

Comment:
Now that this globals is removed, I think the shared namespace can be moved to logic.